README.md

```diff
@@ -1,113 +1,101 @@
     # # # Distribution Statement A. Approved for public release. Distribution unlimited.
-    # # # 
+    # # #
     # # # Author:
     # # # Naval Research Laboratory, Marine Meteorology Division
-    # # # 
-    # # # This program is free software:
-    # # # you can redistribute it and/or modify it under the terms
-    # # # of the NRLMMD License included with this program.
-    # # # 
-    # # # If you did not receive the license, see
+    # # #
+    # # # This program is free software: you can redistribute it and/or modify it under
+    # # # the terms of the NRLMMD License included with this program. This program is
+    # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+    # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+    # # # for more details. If you did not receive the license, for more information see:
     # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
-    # # # for more information.
-    # # # 
-    # # # This program is distributed WITHOUT ANY WARRANTY;
-    # # # without even the implied warranty of MERCHANTABILITY
-    # # # or FITNESS FOR A PARTICULAR PURPOSE.
-    # # # See the included license for more details.
 
-Installation Guide
-==================
+Tropical Cyclone Recentering GeoIPS Plugin
+===========================================
 
-This installation guide has installation steps specific to installing this plugin, including
-the base geoips conda install if not already installed.
+The recenter_tc package is a GeoIPS-compatible plugin, intended to be used within the GeoIPS ecosystem.
+Please see the 
+[GeoIPS Documentation](https://github.com/NRLMMD-GEOIPS/geoips/blob/main/README.md)
```

Comment:
@mindyls Should we be using a relative path here?

Reply:
@cpcamacho This is a link to another repo, so I don't think relative links will work.  I'm going to resolve, feel free to merge this repo once approved and you are happy with the state of the PR.